src/index.html

```diff
@@ -9,10 +9,53 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="styles/style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
```

Comment:
Consider using a more semantic HTML element instead of `<div>`. For example, if this section represents a rating component, a `<section>` or `<nav>` might be more appropriate.